packages/SwingSet/test/test-transcript.js

```diff
@@ -65,7 +69,7 @@ test('transcript-one load', async t => {
     setAllState(s, states[i]);
     cfg.hostStorage = s;
```

Comment:
should this `cfg.hostStorage = s` be removed, replaced by the `{ hostStorage: s}` done below?

Reply:
Yup.